.travis.yml

```diff
@@ -18,6 +18,11 @@ env:
 install:
   - if [ -n "$CLIPPY" ]; then rustup component add clippy-preview; fi
 
+addons:
+  apt:
+    packages:
+      gperf
+
```

Comment:
This was necessary for a while, but I think it should be good even without it now.

Current master seems to build fine without it.